compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEAssignNpuDmaBdIds.cpp

```diff
@@ -105,11 +119,13 @@ LogicalResult assignNpuDmaBdIds(AMDAIE::WorkgroupOp workgroupOp) {
         auto bdIdOp = rewriter.create<AMDAIE::BdIdOp>(rewriter.getUnknownLoc(),
                                                       tileOp, bdId.value());
         rewriter.setInsertionPoint(npuDmaOp);
+        llvm::outs() << "replace target: " << npuDmaOp << "\n";
```

Comment:
Was this intentional ?

Either remove this or add within `LLVM_DEBUG` maybe?

Reply:
Oopsie, thanks for the catch!